src.ts/account.ts

```diff
@@ -118,9 +119,19 @@ export class Account {
         this.printLogs(signedTx.transaction.receiverId, flatLogs);
 
         if (typeof result.status === 'object' && typeof result.status.Failure === 'object') {
-            throw new TypedError(
-                `Transaction ${result.transaction_outcome.id} failed. ${result.status.Failure.error_message}`,
-                result.status.Failure.error_type);
+            // if error data has error_message and error_type properties, we consider that node returned an error in the old format
```

Comment:
let's add a TODO with GH issue link to track removal of temporary stuff (not a blocker for merge)